server/http/controller.go

```diff
@@ -882,7 +882,26 @@ func (c *controller) GetTransactionRun(ctx context.Context, transactionId string
 		return handleDBError(err), err
 	}
 
-	return openapi.Response(http.StatusOK, c.mappers.Out.TransactionRun(run)), nil
+	openapiRun := c.mappers.Out.TransactionRun(run)
```

Comment:
Yes, terrible approach. But I just want to release this to unblock the frontend fast. We can improve this later.